server/odc-service/src/main/java/com/oceanbase/odc/service/collaboration/IndividualOrganizationMigrator.java

```diff
@@ -99,7 +99,6 @@ private ResourceConfig getResourceConfig(DataSource dataSource, Long userId, Lon
         List<String> resourceLocations = new LinkedList<>();
         resourceLocations.add("migrate/common/V_3_2_0_6__iam_permission.yaml");
         resourceLocations.add("migrate/common/V_3_3_0_4__iam_permission.yaml");
-        resourceLocations.add("migrate/common/V_3_4_0_2__iam_permission.yaml");
```

Comment:
You may need to remove all Yaml files which were removed in your last PR, rather than only this `V_3_4_0_2__iam_permission.yaml`

Reply:
The other two file are missing, re-pushing now. A total of three related documents.

Reply:
The other two file are missing, re-pushing now. A total of three related documents.